proto/decentraland/sdk/components/gltf_node_modifiers.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package decentraland.sdk.components;
+
+import "decentraland/sdk/components/common/id.proto";
+import "decentraland/sdk/components/material.proto";
+
+option (common.ecs_component_id) = 1099;
+
+message PBGltfNodeModifiers {
+  message GltfNodeModifier {
+    string path = 1;
```

Comment:
add doc comments please

Reply:
It's nice to see you around these parts again Agus 🤘 !

I just added explainatory comments to the component 👍